server/helpers/emailTemplates.js

```diff
@@ -31,4 +31,34 @@ const verifyEmailMessage = (token) => {
   };
 };
 
-export default verifyEmailMessage;
+/**
+ * @description loginEmailMessage contains the email message template
+ * @param {string} loginUrl - The token generated for the user
+ * @returns {object} - contains the mail message template
+ */
+
+const loginLinkMessage = (loginUrl, token) => {
+  const err = { errors: {} };
+
+  if (token === undefined || token.trim() === '') {
+    err.errors.token = ['please provide a token'];
+  } if (Object.keys(err.errors).lenght > 0) {
```

Comment:
1. Is this an `if` or `else if`? Should it be on this line or a new one?
2. Does this actually work? With that spelling of len...

Reply:
1. It is `if`, moved to a new line.
2. No, it doesn't - this is fixed now.